charts/influxdb2/templates/statefulset.yaml

```diff
@@ -32,16 +32,62 @@ spec:
         {{- end }}
       containers:
         - name: {{ .Chart.Name }}
-          image: "{{ .Values.image.repository }}:v{{ .Chart.AppVersion }}"
+          image: "{{ .Values.image.repository }}:{{ default .Chart.AppVersion .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
-          args:
```

Comment:
extra `args` should be configurable, for example to configure [a log level in `entrypoint.sh`](https://github.com/influxdata/influxdata-docker/blob/368e523/influxdb/2.0/entrypoint.sh#L28-L35)

Reply:
`influxd print-config --key-name log-level` should take this from `INFLUXD_LOG_LEVEL`